### ISSUE TITLE
 KB2533623 breaks navigating nvda with firefox.

### ISSUE THREAD

**Issue by hamzalatif7** [nvaccess/nvda#10748]
### Steps to reproduce:

step 1 download KB2533623 from the Microsoft update website or using windows update.
step 2 once downloaded install it by clicking on the msu file if you got it from windows update and you selected it just press install updates.
step 3 Once it is installed restart the pc.
 ### Actual behavior:
 The Actual behavior that I get instead is nvda  is not able to read a website in firefox. All firefox  Dialog boxes   are red fine such as the address bar, the options  Dialog and any other things that are directly linked to firefox.

### Expected behavior: The Expected behavior:

 would be nvda reading webpages in the firefox browser correctly.
 ### System configuration

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have tried the beta versions of nvda and they fail with the same results.

### Additional information  that may help 
https://www.microsoft.com/en-us/download/details.aspx?id=26764 the link to the updates details and download page just click on the kb article  that is linked there.
This only seems to effect the reading  of webpages  on firefox and nvda however when i use firefox ports such as waterfox it works fine.

**Comment by Adriani90** [comment]
Could you please run the com registration fixing tool in nvda menu / tools? Is the issue still reproducible after that?

**Comment by hamzalatif7** [comment]
I have tried this and i still get the same problem. There are errors in the debug log  that i get when NVDA tries to navigate    a webpage in firefox I can send it if needed. The log contains tracebacks relating to the reading of firefox.

**Comment by Adriani90** [comment]
@hamzalatif7 could you please give an update if this is still a problem with NVDA 2019.3.1 and the last windows 10 updates?
cc: @michaelDCurran, @feerrenrut
